app/views/sufia/admin/admin_sets/index.html.erb

```diff
@@ -27,6 +28,7 @@
                     <td><%= link_to admin_set.title.first, [sufia, :admin, admin_set] %></td>
                     <td><%= admin_set.create_date %></td>
                     <td><%= safe_join(admin_set.creator) %></td>
+                    <td><%= @presenter.total_items(admin_set) %></td>
```

Comment:
I wasn't able to get this to work. Is the idea to remove the need to declare @presenter in the controller? Would additional code be needed to define controller? Replaced as given did not work for me.